views/css/payments.css

```
@@ -20,6 +20,12 @@
   margin-top: .55rem;
 }
 
+.product-add-to-cart #ps-checkout-express-button div.paypal-buttons-context-iframe {
+  margin-top:1.25rem;
+  width: 45%;
+  margin-left:75px;
```

```suggestion
  margin-top: 1.25rem;
  width: 45%;
  margin-left: 75px;
```

Keep consistency inside your css files please, if you don't want to mind about it, you can also configure stylelint to do it for you like inside the PrestaShop main repository